app/src/main/java/com/framgia/music_44/screen/fragmentHome/adapter/AdapterHome.java

```diff
@@ -0,0 +1,76 @@
+package com.framgia.music_44.screen.fragmentHome.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.framgia.music_44.R;
+import com.framgia.music_44.data.model.Songs;
+import com.framgia.music_44.screen.util.OnItemClickListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AdapterHome  extends RecyclerView.Adapter<AdapterHome.ViewHolder>{
+    private List<Songs> mSongsList;
+    private LayoutInflater mLayoutInflater;
+    private Context mContext;
+    private OnItemClickListener mOnItemClickListener;
+
+    public AdapterHome(Context context) {
+        mContext = context;
+        mSongsList = new ArrayList<>();
+    }
+    public void setOnItemClickListener(OnItemClickListener onItemClickListener){
+        mOnItemClickListener = onItemClickListener;
+    }
+    public void updateList(List<Songs> songs){
+        mSongsList.addAll(songs);
+        notifyDataSetChanged();
+    }
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        mLayoutInflater= LayoutInflater.from(viewGroup.getContext());
+        View view = mLayoutInflater.inflate(R.layout.item_list,viewGroup,false);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
+        Glide.with(mContext).load(mSongsList.get(i).getImage()).into(viewHolder.mImageViewSongs);
```

Comment:
tạo 1 method để xử lý riêng phần này
```suggestion
       viewHolder.bindViewData(mSongList.get(i))
```